PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,8 +3,8 @@
 
 ---
 <!-- Thank you for contributing to Servo! Please replace each `[ ]` by `[X]` when the step is complete, and replace `__` with appropriate data: -->
-- [ ] `./mach build -d` does not report any errors
-- [ ] `./mach test-tidy` does not report any errors
+- [X] `./mach build -d` does not report any errors
+- [X] `./mach test-tidy` does not report any errors
```

Comment:
The template should not be pre-checked; you should be making those changes in the description of this pull request, instead.